db/data.py

```diff
@@ -27,6 +27,7 @@
     ['tonal'],
 ]
 
+# TODO: Util methods should not be in the database package
```

Comment:
right, `acousticbrainz.something`. I'm thinking that we should have something like

acousticbrainz/manager/data.py - interface for doing stuff
acousticbrainz/db/data.py - lowlevel database, only called by the manager